db/db_test.cc

```diff
@@ -4119,9 +4137,9 @@ TEST_F(DBTest, WriteSingleThreadEntry) {
   std::vector<port::Thread> threads;
   dbfull()->TEST_LockMutex();
   auto w = dbfull()->TEST_BeginWrite();
-  threads.emplace_back([&] { Put("a", "b"); });
+  threads.emplace_back([&] { ASSERT_OK(Put("a", "b")); });
```

Comment:
I am not sure if this is correct or if it needs to return the Status and someone else has to check it.  I am not sure the semantics of a thread fails/asserts.

Reply:
"ASSERT_OK" just makes sure "Put" returns the correct status. If not, the test case should show the failure.